src/test/java/LoginTests.java

```diff
@@ -8,14 +10,20 @@
 public class LoginTests extends BaseTest {
 
     @Test
-    public static void LoginEmptyEmailPasswordTest () {
+    public void Navigation () {
 
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
         String url = "https://testpro.io/";
         driver.get(url);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
+     
+
+        WebElement registrationLink = driver.findElement(By.cssSelector("[id='hel']"));
+        registrationLink.click();
+
+        String registrationUrl = "https://bbb.testpro.io/registration.php";
+        Assert.assertEquals(driver.getCurrentUrl(), registrationUrl);
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/210998289-d5bae945-562f-4693-a563-76d0c53b6216.png)

![image](https://user-images.githubusercontent.com/104761482/210998330-a38bfed1-2679-4735-af0f-7d5c83079763.png)

Hint

```
        String url = "https://bbb.testpro.io/";
        driver.get(url);
```
     
![image](https://user-images.githubusercontent.com/104761482/210998561-fe81c4ad-bd13-432f-b1de-7a853facfc57.png)